specs/swagger.yml

```diff
@@ -6631,8 +6631,6 @@ definitions:
         properties:
           isRunning:
             type: boolean
-          isSigned:
```

Comment:
any chance this will cause mop a regression of any flavor? i doubt it but figured i'd ask. almost seems 'breaking ish'

Reply:
Nope.